markets/perps-market/storage.dump.sol

```diff
@@ -593,7 +593,6 @@ library PerpsMarket {
         uint128 marketId;
         int256 skew;
         uint256 size;
-        int256 sizeDelta;
```

Comment:
This dump file looks a bit weird to me..Can you doublecheck that it's correct?

Reply:
This is correct actually. I moved this `sizeDelta` into the runtime during order settlement. I think the dump file wasn't updated in the previous PR.

Reply:
Cool thanks for checking.. We should get to the bottom of the dump.sol checker, CI should fail if dump file isn't updated.